### PR TITLE
Change date style in French translations

### PR DESCRIPTION
**Pull request by Walavouchey**
`yyyy-mm-dd` --> `dd/mm/yyyy`

```sh
# change date format `yyyy-mm-dd` --> `dd/mm/yyyy`
for file in $( find wiki -name fr.md )
do
    perl -i -pe 's/(?<!news\/)([0-9]{4})-([0-9]{2})-([0-9]{2})/\3\/\2\/\1/g' $file
done
# fix date ranges `dd/mm/yyyy/dd/mm/yyyy` --> `dd/mm/yyyy - dd/mm/yyyy`
for file in $( find wiki -name fr.md )
do
    perl -i -pe 's/([0-9]{2}\/[0-9]{2}\/[0-9]{4})\/([0-9]{2}\/[0-9]{2}\/[0-9]{4})/\1 - \2/g' $file
done
```

`wiki/News_styling_criteria/fr.md` is excluded as the changes are invalid

<details> <summary>(wysi)</summary>

</details>

### CODE REVIEW

**Review by DoryanR — DISMISSED** [review]
Thank you for this PR. I was trying to do that when I was doing page updates. I really need to learn how to use these python scripts, it looks really useful.

**Comment by Walavouchey** [comment]
this is bash, all on the command line :3

**Review by Biokax — APPROVED** [review]
That a really huge cleanup which saves us hours of rereading date format thanks (you got me this time haha)

File: wiki/Gameplay/Replay/fr.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Un replay exporté aura un fichier sous le format `.osr` avec un nom écrit comm
 
 ```
 Format : {Nom local du joueur} - {Artiste} - {Titre} [{Difficulté}] ({AAAA-MM-JJ}) {GameMode}
-Exemple: dummytest1 - Loituma - Ievan Polkka [SPINNER-MADNESS] (2013-08-12) OsuMania
+Exemple: dummytest1 - Loituma - Ievan Polkka [SPINNER-MADNESS] (12/08/2013) OsuMania
```
Comment (TPGPL) [inline wiki/Gameplay/Replay/fr.md:99]: this should be reverted as its a specific format

Comment (Walavouchey) [reply wiki/Gameplay/Replay/fr.md]: fixed

**Comment by peppy** [comment]
Seems pretty insane to manage this manually per localisation, but I'll turn a blind eye :D.

**Comment by Walavouchey** [comment]
yeah... if only there was a special syntax for automatic date localisation

**Comment by Walavouchey** [comment]
though it wouldn't really be that useful since dates usually tend to not change very much over long periods of time

**Comment by peppy** [comment]
well i mean, i didn't realise french people can't read `yyyymmdd` dates. also is a weird one because this same change should probably be applied to any non-japanese non-american countries as well? and what about the default english then? if you presume that americans also can't read `yymmdd` and want `MMdd` instead then we have a situation on our hands (because I'm not about to take on broken american standards).

File: wiki/Client/Program_files/fr.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Le dossier Replays contient les fichiers replay du joueur. Un fichier replay ne
 
 La structure du nom de fichier est `Nom du joueur local} - {Artiste} - {Titre} {[Difficulté]}{(AAAA-MM-JJ)} {Mode de jeu}`. Un exemple de cette structure est présenté ci-dessous :
 
-``dummytest1 - Loituma - Ievan Polkka \[SPINNER-MADNESS\] (2013-08-12) OsuMania``
+``dummytest1 - Loituma - Ievan Polkka \[SPINNER-MADNESS\] (12/08/2013) OsuMania``
```
Comment (Walavouchey) [inline wiki/Client/Program_files/fr.md:49]:
```suggestion
``dummytest1 - Loituma - Ievan Polkka \[SPINNER-MADNESS\] (2013-08-12) OsuMania``
```

**Comment by DoryanR** [comment]
On our side in France we use the format `dd/mm/yyyy`, I can confirm

**Comment by DoryanR** [comment]
Besides, since some languages may have this problem, why not make a CI about it? This would avoid having to do this work again. Of course, the CI only concerns countries that use this type of format (`dd/mm/yyyy`), without impacting other localizations.

**Comment by Walavouchey** [comment]
Would rather have this automated by special syntax than enforced for each language via CI, but that's way out of scope for now. It's not that hard to fix in build anyway as I hope OP has illustrated.

**Review by DoryanR — APPROVED** [review]

**Review by Biokax — APPROVED** [review]

**Comment by Walavouchey** [comment]
can we get some sort of conclusion on this? if it gets merged, i'll do the same for german for now, and i hope to do other bulk search-and-replace changes like this soon (mainly concerning terminology in tournament articles)

**Review by TicClick — APPROVED** [review]

**Pull request merged**